<?xml version="1.0" encoding="UTF-8"?>
<_ui version="4.0">
    <class>AnimationPreSettings</class>
    <widget class="QWidget" name="AnimationPreSettings">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>400</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>400</width>
                <height>400</height>
            </size>
        </property>
        <property name="baseSize">
            <size>
                <width>400</width>
                <height>400</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Animation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>LED matrix:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QTextEdit" name="editWidth">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>40</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>100</width>
                                    <height>27</height>
                                </size>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>X</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QTextEdit" name="editLength">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>40</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>100</width>
                                    <height>27</height>
                                </size>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="buttonSetWL">
                            <property name="text">
                                <string>Set</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="4" column="0">
                <layout class="QGridLayout" name="gridLayout_2">
                    <item row="0" column="2">
                        <widget class="QComboBox" name="selectMode">
                            <property name="currentIndex">
                                <number>2</number>
                            </property>
                            <item>
                                <property name="text">
                                    <string>Polygon</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>Circle</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>Basic</string>
                                </property>
                            </item>
                        </widget>
                    </item>
                    <item row="0" column="0">
                        <widget class="QLabel" name="label_3">
                            <property name="text">
                                <string>Animation mode:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <spacer name="horizontalSpacer_3">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item row="1" column="0">
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item row="5" column="0">
                <widget class="Line" name="line_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item row="6" column="1">
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>1</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</_ui>
